src/kOS.Safe/Execution/CPU.cs

```diff
@@ -15,15 +15,18 @@ namespace kOS.Safe.Execution
 {
     public class CPU : ICpu
     {
-        private enum Status
+        private enum Section
         {
-            Running = 1,
-            Waiting = 2
+            Main = 1,
+            Trigger = 2
         }
```

Comment:
Should we maybe include an enum for "interpreter".  It might allow us to address #1787 at the same time.  I guess that really isn't what this enum is used for though, so maybe it's a bad idea.  Just a thought that popped into my head.